### DISCUSSION TITLE
Task 1: local tensor zero values and base model array empty

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#54] [General]
Hi 
If I get 0 value in the local_tensor like `local_tensors
[LocalTensor(col_name='01', tensor=array(0., dtype=float32), weight=0.5), LocalTensor(col_name='02', tensor=array(0., dtype=float32), weight=0.5)]` and `base_model_nparray.shape = 
()` Can you tell me what it means?

**Comment by sarthakpati** [comment]
Hi,

I can take an educated guess, but I will wait for @brandon-edwards and/or @psfoley to respond for added clarity.

Cheers,
Sarthak

**Reply by psfoley** [reply]
@dskhanirfan Thanks for the question. The custom aggregation functions are applied layer-wise to the model, and each of `local_tensors` correspond to each of the contributions coming from each of the collaborators who participated in that training round. In your example, it appears there are two collaborators ('01' and '02'), that both have weight values of `0` for the layer being processed (each with weight of 50%). Let me know if any other clarification is needed and I would be glad to expand!
